src/cijobs/cijobs.cs

```diff
@@ -523,22 +534,22 @@ private class CopyCommand
                 Directory.CreateDirectory(outputPath);
 
                 // Pull down the zip file.
-                DownloadZip(cic, config, outputPath).Wait();
+                DownloadZip(cic, config, outputPath, config.ContentPath).Wait();
             }
 
             // Download zip file.  It's arguable that this should be in the 
-            private static async Task DownloadZip(CIClient cic, Config config, string outputPath)
+            private static async Task DownloadZip(CIClient cic, Config config, string outputPath, string contentPath)
             {
                 // Copy product tools to output location. 
-                bool success = cic.DownloadProduct(config, outputPath).Result;
+                bool success = cic.DownloadProduct(config, outputPath, contentPath).Result;
 
                 if (config.DoUnzip)
                 {
                     // unzip archive in place.
-                    var zipPath = Path.Combine(outputPath, "Product.zip");
+                    var zipPath = Path.Combine(outputPath, Path.GetFileName(contentPath));
                     ZipFile.ExtractToDirectory(zipPath, outputPath);
                 }
             }
         }
     }
-}
+}
```

Comment:
This is a nit, but could you add CR here so there isn't the little git nag about it being missing?